sdk-api-src/content/synchapi/nf-synchapi-releasesrwlockexclusive.md

```diff
@@ -70,6 +70,12 @@ Releases a slim reader/writer (SRW) lock that was acquired in exclusive mode.
 A pointer to the SRW lock.
 
 
+## -remarks
+
+SRW lock cannot be released by a different thread from the one that has acquired it.
+Correct thread usage may not be enforced by the API, it is undefined what happens on violations.
```

Comment:
I would avoid talking about implementation details. Talk about the contract. I'd also rephrase to avoide the double-negative

> The SRW lock must be released by the same thread that acquired it. You can use Application Verifier to help verify that your program uses SRW locks correctly.

Similar remarks apply to ReleaseSRWLockShared.

Reply:
Changed to the suggested text in both functions (with also directly specifying which Application Verifier checker is used for that)